app/main.py

```diff
@@ -2,4 +2,74 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list[int] = None
+                 ) -> None:
+        if coords is None:
+            coords = [0, 0]
+        self.name = name
+        self.weight = weight
+        self.coords = coords
```

Comment:
```suggestion
        self.coords = coords or [0, 0]
```
this one will be better. Also, by doing this you can delete the `if-statement` above. Fix everywhere.